site/layouts/shortcodes/orange-footer.html

```diff
@@ -64,7 +64,7 @@ <h3 class="footer-heading">Follow us</h3>
       <div class="row">
         <div class="footer-column col-md-3">
           <h3 class="accordion-header footer-heading" id="headingOne{{ $id }}">
-            <button class="accordion-button collapsed container-xxl px-1 d-md-none" type="button" data-bs-toggle="collapse" data-bs-target="#collapseOne{{ $id }}" aria-expanded="true" aria-controls="collapseOne{{ $id }}">
+            <button class="accordion-button collapsed container-xxl px-1 px-md-2 d-md-none" type="button" data-bs-toggle="collapse" data-bs-target="#collapseOne{{ $id }}" aria-expanded="true" aria-controls="collapseOne{{ $id }}">
```

Comment:
Are those `.px-md-2` useful given the fact that we have `.d-md-none`?

Reply:
It is if people use `.navbar-expand-lg` for example but I see your point and I think it can be removed for our examples